zbsprimarystorage/zbsprimarystorage/zdaemon.py

```
@@ -23,7 +23,7 @@ def prepare_pid_dir(path):
 def main():
     usage = 'usage: python -c "from zbsprimarystorage import zdaemon; zdaemon.main()" start|stop|restart'
     if len(sys.argv) != 2 or not sys.argv[1] in ['start', 'stop', 'restart']:
-        print
+        print()
         usage
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)